website/docs/r/transfer_user.html.markdown

```diff
@@ -73,8 +73,8 @@ The following arguments are supported:
 
 * `server_id` - (Requirement) The Server ID of the Transfer Server (e.g. `s-12345678`)
 * `user_name` - (Requirement) The name used for log in to your SFTP server.
-* `home_directory` - (Optional) The landing directory (folder) for a user when they log in to the server using their SFTP client.
-* `policy` - (Optional) An IAM JSON policy document that scopes down user access to portions of their Amazon S3 bucket. IAM variables you can use inside this policy include `${Transfer:UserName}`, `${Transfer:HomeDirectory}`, and `${Transfer:HomeBucket}`. Since the IAM variable syntax matches Terraform's interpolation syntax, they must be escaped inside Terraform configuration strings (`$${Transfer:UserName}`).
+* `home_directory` - (Optional) The landing directory (folder) for a user when they log in to the server using their SFTP client.  It should begin with a `/`.  The first item in the path is the name of the homeBucket. (e.g. `/example-bucket-1234/username` would set the homeBucket to `example-bucket-1234` and the homeDirectory to `username`.  This allows the use of the variables descripted in the scope-down policy.
```

Comment:
I think we should clarify what `homeBucket` and `homeDirectory` mean here. As written they seem like specific Transfer terminology. My recommendation is either making them normal words and/or referencing their equivalent variable names. If they do have some special meaning written homeBucket/homeDirectory, then it might be worth clarifying that with a link reference.

```suggestion
* `home_directory` - (Optional) The landing directory (folder) for a user when they log in to the server using their SFTP client.  It should begin with a `/`.  The first item in the path is the name of the home bucket (accessible as `${Transfer:HomeBucket}` in the policy) and the rest is the home directory (accessible as `${Transfer:HomeDirectory}` in the policy). For example, `/example-bucket-1234/username` would set the home bucket to `example-bucket-1234` and the home directory to `username`.
```